test/junit/scala/collection/immutable/ListMapTest.scala

```diff
@@ -0,0 +1,16 @@
+package scala.collection.immutable
```

Comment:
Confirming this was an assumption I had made. Please wait for the official policy.